pkg/framework/aries/default.go

```diff
@@ -19,6 +19,12 @@ func (d defFramework) transportProviderFactory() api.TransportProviderFactory {
 	return transport.NewProviderFactory()
 }
 
+// transportProviderFactory provides default Outbound Transport provider factory
+func (d defFramework) didResolverProviderFactory() DIDResolver {
```

Comment:
I might have had this comment previously but I'm not sure that we need the two word suffix of "ProviderFactory". Perhaps you could elaborate on why we need more than "Provider"?

Reply:
@troyronda my bad. I had missed to update this. Updated this in a [new patch](https://github.com/hyperledger/aries-framework-go/compare/503cb40534259c3739b5647d3d3c93464307993d..f0888dc86951e726e7afc54054ddd0fba13b7705#diff-da8e5280b0328cb616f0b42390a35b00R23).